library/ssl_client.c

```diff
@@ -416,40 +416,35 @@ static int ssl_write_sig_alg_ext( mbedtls_ssl_context *ssl, unsigned char *buf,
 /* Write cipher_suites
  * CipherSuite cipher_suites<2..2^16-2>;
  */
-/**
- * \brief Validate cipher suite against config in SSL context.
- *
- * \param ssl         SSL context
- * \param suite_info  Cipher suite to validate
- *
- * \return 0 if valid, else 1
- */
-static int ssl_validate_ciphersuite(
+int mbedtls_ssl_validate_ciphersuite(
```

Comment:
Perhaps pre-existing: the comment above, "write cipher_suite" looks like it should go with `ssl_write_client_hello_cipher_suites()` instead than remain here.